### PR TITLE
[Component with its own class name as service type] LPS-199467 Refactor Helper and importer class from wiki module

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-199467

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:wiki

**Comment by catarineaguiar** [comment]
no regressions found

File: modules/apps/wiki/wiki-web/src/main/java/com/liferay/wiki/web/internal/base/BaseMVCActionCommand.java [hunk 1]
```diff
@@ -1,9 +1,9 @@
 /**
- * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-package com.liferay.wiki.web.internal.helper;
+package com.liferay.wiki.web.internal.base;
```
Comment (rafaprax) [inline modules/apps/wiki/wiki-web/src/main/java/com/liferay/wiki/web/internal/base/BaseMVCActionCommand.java:6]: Hey @MrJohn1911, move it to the `portlet.action` package.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MrJohn1911** [comment]
ci:test:sf

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:stable

**Comment by zhangyan0701** [comment]
ci:test:wiki

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
The last change in this pull was just an update in the bnd.bnd file

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
ci:test:wiki

**Review by dantewang — APPROVED** [review]

**Comment by tinatian** [comment]
ci:forward

**Pull request closed without merging**